Make the description of issue templates easier (#4168)
files-community/Files: .github/ISSUE_TEMPLATE/bug_report.md

<!-- 
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
I ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING:
1. Issue may be deleted if it is not following the template
2. Only post one issue per one bug report
3. Try not to make duplicates. Do a quick search before posting

> 4. Add a clarified title

Reviewer: ```suggestion
4. Add a title that describes the issue clearly
```

--->

**Explain the bug**
Write a clear description of what the bug is.

**To reproduce**
1. Go to '....'
2. Click on '....'
3. Scroll down to '....'
4. See the error

**Expected behavior**
Write a clear description of what you expected.

**Logfile**
Upload the log file here for us to understand your issue better. 
- You can access it from Settings->About->Open log location. 
or 

> - Copy the debug.txt and debug_fulltrust.txt in the following directory, and paste it here.

Reviewer: ```suggestion
- Copy the debug.txt and debug_fulltrust.txt from the following directory, and upload the files here.
```
